lib/src/themes/color_parser.dart

```diff
@@ -78,9 +93,9 @@ class ColorParser {
         final g = int.tryParse(components[1]);
         final b = int.tryParse(components[2]);
         final alpha =
-            components.length == 3 ? 1.0 : double.tryParse(components[3]);
+            components.length == 3 ? 1.0 : alphaValueToDouble(components[3]);
         if (r != null && g != null && b != null && alpha != null) {
-          return Color.fromARGB((0xff * alpha).toInt(), r, g, b);
+          return Color.fromARGB((0xff * alpha).round().toInt(), r, g, b);
```

Comment:
Subtle change here: the previous behavior was to truncate, which will result in a lot of alpha values (like 36%) being off by one. The RGBA tests verify correctness and should ensure against regression as-written.